torch/_torch_docs.py

```diff
@@ -8292,25 +8292,30 @@ def merge_dicts(*dicts):
 svd(input, some=True, compute_uv=True, *, out=None) -> (Tensor, Tensor, Tensor)
 
 Computes the singular value decomposition of either a matrix or batch of
-matrices :attr:`input`. The singular value decomposition is represented as a
-namedtuple `(U, S, V)`, such that :attr:`input` `= U diag(S) Vᴴ`.
+matrices :attr:`input`.
+
+The singular value decomposition is represented as a
+namedtuple `(U, S, V)`, such that `input = U diag(S) Vᴴ`.
 where `Vᴴ` is the transpose of `V` for real inputs,
 and the conjugate transpose of `V` for complex inputs.
-If :attr:`input` is a batch of matrices, then `U`, `S`, and `V` are also
+If :attr:`input` is a batch of matrices, then `U`, `S`, `V` are also
 batched with the same batch dimensions as :attr:`input`.
 
+The singular values are returned in descending order. If :attr:`input` is a batch of matrices,
+the singular values of each matrix in the batch are returned in descending order.
+
 If :attr:`some` is `True` (default), the method returns the reduced singular
-value decomposition. In this case, if the last two dimensions of :attr:`input` are
-`m` and `n`, then the returned `U` and `V` matrices will contain only
-`min(n, m)` orthonormal columns.
+value decomposition. In this case, if :attr:`input` is of size `(*, m, n)`,
+the returned `U`, `V` will be of size `(*, m, min(m, n))` and `(*, n, min(m, n))`
```

Comment:
"then \`U\` will have shape ..., and \`V\` will have shape..."

Reply:
Applied the rule "prefer shape over size" to the documentation.

Reply:
Also prefer "has shape XX" to "is of shape XX"